arbitrum-docs/partials/_glossary-partial.md

```diff
@@ -1,338 +1,437 @@
 
-
 ### Active Validator {#active-validator}
+
```

Comment:
Where do these changes come from? If I generate the glossary and format it, I don't have these new lines.